docs/leo-rover/integrations/cameras/unispectral-monarch-pro.mdx

```diff
@@ -0,0 +1,149 @@
+
+### Installing the packages
+
+First you have to download it into `src` directory of your ROS workspace:
+
+```bash
+cd <path_to_your_ros_workspace>/src
+git clone https://github.com/fictionlab/ros_monarch_pro.git
+```
+
+Having the source code, we need have installed all the required dependencies so
+the package will be installed successfully. To install the dependencies type:
+
+```bash
+source devel/setup.bash
+rosdep install --from-paths src -iry
+```
+
+:::note
+
+If this is your first package in the ROS workspace, or you haven't built it yet
+you need to source `/opt/ros/<your distro>/setup.bash`
+
+:::
+
+Now to install the package, you just have to type:
+
+```bash
+catkin build
+```
+
+If the building process was successful, you are ready to run the wrapper.
+
+## Examples
+
+### Running the node
+
+Running the installed software is really easy as we have prepared launch file
+for it. To run it, enter your ROS workspace and type:
+
+```bash
+source devel/setup.bash
+roslaunch monarch_ros wrapper.launch
+```
+
+The node provides a ROS service, which saves a spectral cube into images to
+specified directory. To trigger the image save you need to run in other terminal
+(with your ROS workspace sourced):
+
+```bash
+rosservice call /monarch_pro_wrapper/save_images
+```
+
+If the save was successful you will receive the path to parent directory, where
+the data is stored. Each call to the service results in creation of a separate
+directory named like `cube_<date>` inside the parent directory, which contains
+the saved files. By default the parent directory is set to your home directory.
+To change it you can either change the argument inside the `wrapper.launch`
+file, or you can set its value during launching the program - for example:
+
+```bash
+roslaunch monarch_ros wrapper.launch data_path:="/<your>/<desired>/<path>"
+```
+
+### Configuring the node
+
+In the downloaded repository, there is a config file with camera parameters. The
+parameters are loaded to the node automatically during the launch process. You
+can change the provided camera parameters to your desired values (available
+settings are `gain` and `exposure fps`). To load them into the program and your
+camera, you need to restart the node.
+
+If you have ROS installed on your computer, you can also change the camera image
+with rqt plugin - dynamic reconfigure - which doesn't require node restart.
+
+Before starting any of the programs, make sure you completed the **Connecting
+another computer to ROS network** of **ROS development** tutorial:
+
+<LinkButton docId="leo-rover/advanced-guides/ros-development" />
+
+Now you can start `rqt`, and from `plugins` -> `configuration` choose
+`dynamic reconfigure` and then choose the `monarch_pro_wrapper` node. On the
+right side of the screen you will see the mentioned parameters - you can use the
+sliders and boxes next to them to change the values of the parameters.
+
```

Comment:
Add a section about adding the node to autostart in `/etc/ros/robot.launch`

Reply:
I think it's redundant right now.